### PR TITLE
SPCTR-1533 UAG Number Control - For offset by and post per page option.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1533

### Screenshots

- https://share.getcloudapp.com/E0uy1QEp

### Types of changes

- Improvement.

### How has this been tested?

- Added in the card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/post/post-carousel/edit.js [hunk 1]
```diff
@@ -638,20 +639,20 @@ const UAGBPostCarousel = ( props ) => {
...
 					value={ postsToShow }
 					data={ {
 						value: postsToShow,
 						label: 'postsToShow',
 					} }
-					setAttributes={ setAttributes }
-					displayUnit={ false }
 					min={ 1 }
 					max={ 100 }
+					displayUnit={ false }
```
Comment (RajkiranBagal) [inline src/blocks/post/post-carousel/edit.js:655]: Remove unwanted props passed to UAGNUmberControl, check all files.

File: src/components/number-control/editor.lazy.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
+.uag-number-control {
+	.uagb-number-control__mobile-controls {
+		align-items: center;
+		display: grid;
+		grid-template-columns: 75% 25%;
+		color: #50575E;
```
Comment (RajkiranBagal) [inline src/components/number-control/editor.lazy.scss:6]: Use variable for applying color

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/blocks/post/post-carousel/edit.js (hunk 1 above)
Comment (AbhijitNage123) [reply src/blocks/post/post-carousel/edit.js:655]: display unit and other props may be needed in future, So keeping it same

File: src/components/number-control/editor.lazy.scss (hunk 2 above)
Comment (AbhijitNage123) [reply src/components/number-control/editor.lazy.scss]: Resolved

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**